packages/components/src/components/badges/badge.tsx

```diff
@@ -0,0 +1,40 @@
+import { Component, Prop, h } from '@stencil/core';
+import { CssClassMap } from '../../utils/utils';
+import classNames from 'classnames';
+
+@Component({
+    tag: 't-badge',
+    styleUrls: [
```

Comment:
we are using just one css file currently instead of 'styleUrls: [...]' you can use 'styleUrl: filename.css'

Reply:
solved